Common/src/main/java/mezz/jei/common/util/TagUtil.java

```diff
@@ -19,9 +19,6 @@ public static <VALUE, STACK> Optional<ResourceLocation> getTagEquivalent(
 		Function<STACK, VALUE> stackToValue,
 		Supplier<Stream<Pair<TagKey<VALUE>, HolderSet.Named<VALUE>>>> tagSupplier
 	) {
-		if (stacks.size() < 2) {
```

Comment:
As strange as it may sound, some of the tags provided by minecraft do have only one item

Reply:
Is it useful for players to see tags that have only one item?
I put this filter because I assumed it was not useful, but I can reconsider!
Maybe it can be a config option.

Reply:
I think it would be better to display it if the recipe used the tag

Reply:
![Screenshot 2024-07-12 at 1 29 49 PM](https://github.com/user-attachments/assets/1a012e9c-560e-43ac-9b5a-ba78b11300ec)
I found some weird tags that are not useful to see, so I will make it configurable